source/code-snippets/usage-examples/changeStream.js

```diff
@@ -5,6 +5,11 @@ const uri = "<connection string uri>";
 
 const client = new MongoClient(uri);
 
+const simulateAsyncPause = () =>
+  new Promise(resolve => {
+    setTimeout(() => resolve(), 100);
```

Comment:
```suggestion
    setTimeout(() => resolve(), 1000);
```
Question: Why was the time lowered? Is that enough time? I think this may make this harder to talk about later on.